axelrod/strategies/stalker.py

```diff
@@ -41,38 +43,41 @@ class Stalker(Player):
 
     def __init__(self) -> None:
         super().__init__()
+        self.R, self.P, self.S, self.T  = self.match_attributes["game"].RPST()
         self.scores = []
-        (R, P, S, T) = self.match_attributes["game"].RPST()
-        very_good_score = R
-        very_bad_score = P
-        wish_score = (P + R) / 2
 
     def score_last_round(self, opponent: Player):
+        # Load the default game if not supplied by a tournament.
         game = self.match_attributes["game"]
         last_round = (self.history[-1], opponent.history[-1])
         scores = game.score(last_round)
         self.scores.append(scores[0])
 
-    def strategy(self, opponent: Player)-> Action:
-        # Update scores from the last play
+    def strategy(self, opponent: Player) -> Action:
+
+        if len(self.history) == 0:
+            return C
+
         self.score_last_round(opponent)
 
-        current_average_score = sum(scores) / len(scores)
+        current_average_score = sum(self.scores) / len(self.scores)
```

Comment:
It's better to keep a cumulative score total then to save the scores for each round unless you need the individual scores. Summing the list each time is O(n^2) in the length of the match, keeping a cumulative sum is O(n).